### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by subtank**
Chinese translation for readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +53,16 @@ This plugin is in Beta. Live support is available for the latest builds through
...
 
+## 依赖插件
+
+- **必须插件**
+   - [Sponge](https://www.spongepowered.org/downloads) - 1.10.2-2281-5.2.0-BETA-2519+
+   - [Grief Prevention](https://forums.spongepowered.org/t/griefprevention-official-thread/1123) - 1.10.2-4.0.0.332+
+   - Permission Plugin - 推荐使用[LuckPerms](https://forums.spongepowered.org/t/luckperms-an-advanced-permissions-plugin/14274)
+- **可选插件**（译者注：这里多推荐一个经济类插件，以便支持岛屿的经济扩展，仅支持API5.0/6.0 here give one econ plugin for sport economy,only for API5.0/6.0）
+   - [Nucleus](https://nucleuspowered.org) - 1.0.1-LTS+
+   - [Total Economy](https://ore.spongepowered.org/Erigitic/Total-Economy)
```
Comment (Mohron) [inline README.md]: Why was this added to the list? I don't even have an Economy support at this time.

File: README.md [hunk 2]
```diff
@@ -21,6 +29,21 @@ This plugin is in Beta. Live support is available for the latest builds through
...
+- [X] 自动创建空岛以及周围领地范围
+- [X] 利用Sponge的schematics文件支持多种预设的空岛模板
+- [X] 可扩展空岛领地范围（需要付出一些资源/金钱需要经济插件支持）
+- [X] 可单独创建一个空岛世界与其他世界分离
+- [X] 自定义岛屿重生点或者tp点
+- [X] 利用GriefPrevention的/trust命令来创建一个空岛队伍
+- [X] 可自定义修改生物群系，小到一个方块，大到你的全岛，随你定义
+- [X] 可以定义限制刷怪机制
+- [X] 自动移除非活跃岛屿
+- [ ] 可配置的岛屿布局格式 (linear & spiral planned)
```
Comment (Mohron) [inline README.md]: Is there a reason you did not translate inside the parenthesis?

File: README.md [hunk 3]
```diff
@@ -6,6 +6,14 @@ With this design, nearly every GriefPrevention feature will be available to play
 
 This plugin is in Beta. Live support is available for the latest builds through [Discord](https://discord.gg/EkVQycV)!
 
+# SkyClaims
+
+SkyClaims是一个运行在Sponge服务端并且依赖于[GriefPrevention](https://forums.spongepowered.org/t/griefprevention-official-thread/1123)保护的空岛服务器插件。
+SkyClaims利用在Sponge服务端上非常棒的一款领地插件GriefPrevention来生成领地保护你的空岛。
+正因为这个特性使得在GriefPrevention上面的几乎所有功能都能在玩家创建的空岛中使用。
+
+本插件还在beta阶段，最新版本的支持在这里[Discord](https://discord.gg/EkVQycV)！
```
Comment (Mohron) [inline README.md:7]: There's no translation for beta??? What!

**Review by Mohron — COMMENTED** [review]
I think in this case I would prefer to have this in it's own file ie. README.zh-CN.md or whatever the appropiate language code is for the Chinese you are translating to. Then I will link them to each other.

File: README.md (hunk 3 above)
Comment (subtank) [reply README.md:7]: actually， in chinese the beta can just use the english~or 贝塔(read those words in beta, LOL)测试 test

File: README.md (hunk 2 above)
Comment (subtank) [reply README.md]: i didn't actually know your means inside the parenthesis.because google it can't read....

File: README.md (hunk 1 above)
Comment (subtank) [reply README.md]: - [X] Expanding islands using claim blocks or other resource (econ)
this line? didnt it means your support Economy ?

File: README.md (hunk 2 above)
Comment (Mohron) [reply README.md]: Linear as in the islands will be placed in a line (how most skyblock plugins did it)
Spiral is how Skyclaims currently does it, it's described in the wiki.

File: README.md (hunk 3 above)
Comment (Mohron) [reply README.md:7]: Was just a surprise is all.

Comment (subtank) [reply README.md:7]: in Chinese, a lot of words can just use the English one and maybe all of us can read.  :）

File: README.md (hunk 2 above)
Comment (subtank) [reply README.md]: well, just replace as this words is OK.
（直线型排列或者螺旋状排列空岛，目前为螺旋状，详细请看[wiki空岛生成相关](https://github.com/DevOnTheRocks/SkyClaims/wiki/%E7%A9%BA%E5%B2%9B%E7%94%9F%E6%88%90%E7%9B%B8%E5%85%B3%EF%BC%88Island-Layout%EF%BC%89)）

**Review by subtank — COMMENTED** [review]
i think you can just copy and paste the Chinese part and upload a new file.i pull this request is just for my translation more easier. lazy to change windows again and again.

File: README.md (hunk 1 above)
Comment (Mohron) [reply README.md]: I haven't gotten around to implementing that ha ha...

Comment (Mohron) [reply README.md]: Only claim blocks are supported at this time.

Comment (subtank) [reply README.md]: well...→_→!!

Comment (Mohron) [reply README.md]: I'm a slacker, I know... I'll fix it on the English version to make it known that the Economy support is upcoming.

Comment (subtank) [reply README.md]: don't update too quickly,i'm slacker too.

**Comment by Mohron** [comment]
https://github.com/DevOnTheRocks/SkyClaims/blob/readme/README.md

File: README.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ SkyClaims利用在Sponge服务端上非常棒的一款领地插件GriefPreventio
 - [X] 可自定义修改生物群系，小到一个方块，大到你的全岛，随你定义
 - [X] 可以定义限制刷怪机制
 - [X] 自动移除非活跃岛屿
-- [ ] 可配置的岛屿布局格式 (linear & spiral planned)
+- [ ] 可配置的岛屿布局格式 (线性或是螺旋状分布岛屿，详见wiki部分)
```
Comment (subtank) [inline README.md]: sorry about this line i forget change

File: README.md (hunk 1 above)
Comment (Mohron) [reply README.md]: Too late ⏩

Comment (subtank) [reply README.md]: will, your means i need retranslation again? GG...in Chinese it is 11pm now...

Comment (Mohron) [reply README.md]: There's no rush!

**Comment by Mohron** [comment]
Superseded by #57
